services/Ebay.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Ebay",

```

Comment:
As described in https://github.com/ambanum/CGUs/issues/106#issuecomment-681870747 (will soon be consolidated in the documentation), we preserve provider casing, both in the name and the ID (file name) 🙂
```suggestion
    "name": "eBay",
```

The file name should also be updated.

![Capture d’écran 2020-10-15 à 15 12 03](https://user-images.githubusercontent.com/222463/96130891-cb466a80-0ef8-11eb-98e5-d0e10ca37dea.png)